packages/editor/src/store/selectors.js

```diff
@@ -1944,7 +2085,7 @@ export const getInserterItems = createSelector(
 	},
 	( state, rootClientId ) => [
 		state.blockListSettings[ rootClientId ],
-		state.editor.present.blocks,
+		state.editor.present.blocks.byClientId,
```

Comment:
I think we probably need `.order` here as well right?

Reply:
Actually, I'm not certain

Reply:
From what I can tell, block order should affect inserter items, but I could be wrong. Is there someone with more domain-specific knowledge of inserter items that could weigh in?

Reply:
Nested blocks might have some impact on that, but as noted in the other PR, I would love us to explore whether we can skip it here.

Reply:
I'm not too concerned with the performance implications of depending on `.order`, from what I've seen, so I'll add in that dependency just to be on the safe side. If this were to produce any issues due to the intricate dependencies you're describing, it could get extremely difficult to debug, so it's not worth the risk.

Reply:
> so I'll add in that dependency just to be on the safe side.

It still needs to be added?

Reply:
Had forgotten to push my changes.